### PR TITLE
Frontend aopp styling update

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

**Comment by thisconnect** [comment]
ok if we squash everything?

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "signing": "To proceed, confirm message on your BitBox02.",
     "success": {
       "message": "Proceed on {{host}}",
-      "title": "Successfully sent address to external service."
+      "title": "Message successfully confirmed and sent"
```
Comment (benma) [inline frontends/web/src/locales/en/app.json]: :-1: again i don't think the "message" is important at all. why not `Address successfully sent`?

File: frontends/web/src/locales/en/app.json [hunk 2]
```diff
@@ -72,7 +72,7 @@
     "signing": "To proceed, confirm message on your BitBox02.",
     "success": {
       "message": "Proceed on {{host}}",
-      "title": "Message successfully confirmed and sent"
+      "title": "Address request confirmed and sent"
```
Comment (benma) [inline frontends/web/src/locales/en/app.json:75]: Better but I would still just do `Address successfully sent` (it is not the address request that is sent).

**Review by benma — APPROVED** [review]
nice! but please change the message :)

**Review by benma — APPROVED** [review]
not sure why i have multiple reviews going on in one PR - maybe cause I went commit by commit...

File: frontends/web/src/locales/en/app.json (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json]: 👍    changed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/488fafe2a809ff7687c60473e0f8f086105418f9..d958cc53580c66825c8208c4fbb95530c7d53df1

**Pull request merged**